backup/backup.go

```diff
@@ -59,6 +67,10 @@ func Backup(rawPaths []string, serviceCh UploadServiceFactory) {
 	if config.Config().UploadStatusInterval != -1 {
 		go func() {
 			for {
+				uploading := stats.CurrentlyUploading()
+				if len(uploading) > 0 {
+					log.Println("Currently uploading:", KeysString(uploading))
```

Comment:
This may panic with "concurrent map writes" because the map is iterated in KeysString which runs outside the stats.lock mutex

Reply:
Probably the code in KeysString should just be moved to CurrentlyUploading

Maybe CurrentlyUploading should return a []string, then this code could `strings.Join` it, that would work safely I think.